builtin/credential/chef-node/backend.go

```diff
@@ -0,0 +1,44 @@
+package chefNode
```

Comment:
The only reason I didn't name it just chef is because you could in theory have a backend that authenticated users against a chef server instead of nodes.  I don't know if anyone would actually want that, but I didn't want to preclude the possibility.